src/psyclone/tests/psyir/frontend/fparser2_test.py

```diff
@@ -423,6 +427,125 @@ def test_process_declarations_intent(f2008_parser):
         Symbol.Access.READWRITE
 
 
+def test_process_declarations_kind_new_param(f2008_parser):
+    ''' Test that process_declarations handles variables declared with
+    an explicit KIND parameter that has not already been declared. '''
+    from fparser.two.Fortran2003 import Specification_Part
```

Comment:
Move import to top of file and remove duplication of import from new tests.

Reply:
Done.